### DISCUSSION TITLE
What does the 3D Game Mode setting do?

### DISCUSSION THREAD

**Discussion by LOuroboros** [Blinue/Magpie#445] [Q&A] [answered]
Title.

I noticed this setting sometime ago inside Magpie's configuration window, but I don't really know what does it do, and since this discussion board exists, I may as well use it to ask about it :eyes: ...

![Magpie_20221028_104049057](https://user-images.githubusercontent.com/4485172/198618043-7f0e903a-539a-4f7f-a835-d44ba70c5fa8.png)

**Comment by hooke007** [comment, marked as answer]
https://github.com/Blinue/Magpie/releases/tag/v0.9.0

> “在 3D 游戏中限制光标”已升级为“3D 游戏模式”，启用时将针对 3D 游戏进行优化，游戏内覆盖也有不同的表现。

https://github.com/Blinue/Magpie/issues/321#issuecomment-1056342891

**Reply by LOuroboros** [reply]
> https://github.com/Blinue/Magpie/releases/tag/v0.9.0
> 
> > “在 3D 游戏中限制光标”已升级为“3D 游戏模式”，启用时将针对 3D 游戏进行优化，游戏内覆盖也有不同的表现。
> 
> [#321 (comment)](https://github.com/Blinue/Magpie/issues/321#issuecomment-1056342891)

So, what I gather from Google Translate is that it optimizes MagPie's upscaling functionality for 3D Games or something?

That's not super descriptive, but it sounds like an option there's no reason not to enable, so uh, thanks!

**Comment by Blinue** [comment]
By default the cursor is optimized for 2D games. Turn this option on for 3D gaming, but disables the following features:
1. multi-screen support
2. move the cursor to other windows (e.g. Start menu)
